pytorch_lightning/trainer/states.py

```diff
@@ -28,14 +29,25 @@ class TrainerState(str, Enum):
     True
     >>> # which is case sensitive
     >>> TrainerState.FINISHED == 'finished'
-    False
+    True
     """
     INITIALIZING = 'INITIALIZING'
     RUNNING = 'RUNNING'
     FINISHED = 'FINISHED'
     INTERRUPTED = 'INTERRUPTED'
 
 
+class RunningStage(LightningEnum):
```

Comment:
Can you refactor and remove

https://github.com/PyTorchLightning/pytorch-lightning/blob/d583d561690ea975ebd9ea41e1b7054495aa0bcb/pytorch_lightning/trainer/connectors/logger_connector/epoch_result_store.py#L23-L41

to use this class?